### DISCUSSION TITLE
Unable to load pre-flight

### DISCUSSION THREAD

**Discussion by MynameisStevetoo** [snipe/snipe-it#15274] [Q&A]
I have had a look through previous questions, and most seem to point to an Apache configuration issue; however I have done all the Apache configuration suggested as fixes, I have checked the .htaccess. 

So I'm running Ubuntu 24.04 managed by Plesk. I'm pulling from git and have followed the instructions fully, including all dependencies etc. 

I'm given a 403 error when I try and access the site to kick off pre-flight, this is in the Apache log:

`AH01276: Cannot serve directory /var/www/vhosts/domain.name/httpdocs/subdomain.domain.name/: No matching DirectoryIndex (index.html,index.cgi,index.pl,index.php,index.xhtml,index.htm,index.shtml) found, and server-generated directory index forbidden by Options directive`

So I have tweaked all the Apache settings to include each DirectoryIndex variant, I have checked the Indexes settings in the .htaccess; I have tried disabling mod_rewrite in case that was messing with anything. I have tried going direct to the /public/ directory, it shows 404 after changing the url to a /setup/ directory url. 

This is my first time pulling from git also, so I'm pulling straight into the root of the subdomain, is that correct?

Apologies if I'm missing something obvious, if anyone has any advice or guidance I'd be hugely appreciative.

**Comment by MynameisStevetoo** [comment]
Ok I changed the document root and got mod_security errors, so disabled mod-security and pre-flight works. So I'll work backwards from there! Sorry for wasting anyone's time that's read this!

**Reply by snipe** [reply]
No worries, glad you got it sorted!
